### ISSUE TITLE
[ACCESSIBILITY] Radio Button invalid state features low contrast colors.

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-css#251]
### Expected Behavior
The color for the invalid state of a Checkbox should be the sam as the invalid state of a Radio Button and both reds should be rated as "accessible" in relation to their contrast with the background.

### Actual Behavior
https://opensource.adobe.com/spectrum-css/2.13.0/docs/?color=light&scale=medium&method=standalone#radio is lighter than https://opensource.adobe.com/spectrum-css/2.13.0/docs/?color=light&scale=medium&method=standalone#checkbox and thus registers a 4.27:1 vs a 4.5:1 contrast which lists as "not accessible" in automated testing.

### Reproduce Scenario (including but not limited to)
I want to use a Radio button in my application.

#### Steps to Reproduce
Use the provided CSS to create a checkbox and a radio button, compare with each other and test against accessibility based contrast requirements.

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)
All

#### Spectrum-CSS version
2.13.0

#### Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/1156657/63528946-dad34a80-c4d1-11e9-91fb-bf8b2097144f.png)
![image](https://user-images.githubusercontent.com/1156657/63529319-7bc20580-c4d2-11e9-9edc-f7d8571b5ae5.png)
![image](https://user-images.githubusercontent.com/1156657/63528981-ecb4ed80-c4d1-11e9-9cfd-844d3e639d98.png)
![image](https://user-images.githubusercontent.com/1156657/63529372-8aa8b800-c4d2-11e9-8ec5-417cde64696c.png)

**Comment by lazd** [comment]
@NateBaldwinDesign can you check this out?

**Comment by NateBaldwinDesign** [comment]
The output of different colors appears to be an error in Spectrum-DNA's output, so I will create a ticket to investigate what is going on there.

As for the contrast minimum requirement, I will have to follow up on the interpretation of the guideline in this scenario.

**Comment by NateBaldwinDesign** [comment]
Ok follow up w/r/t contrast, this situation has a minimum contrast requirement of 3:1 with the background, which these **colors both exceed**: https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by lazd** [comment]
Thanks @NateBaldwinDesign. We can keep this issue open to address the difference in color, but it sounds like we're good with accessibility.

**Comment by Westbrook** [comment]
Bump that the v3 packages still exhibit this difference. When these do match, do you know which of the colors would be preferred? I can apply that in the Spectrum Web Components CSS while this works its way through the DNA priority pipeline.

**Comment by GarthDB** [comment]
Looks like the token is in there and we will release an update with @lazd work on the radio button.
